docs/reference/concepts/code-generation.md

```diff
@@ -10,32 +10,51 @@ The code generator runs when you do one of the following:
 * Select **SpatialOS** > **Generate Code** from the Unity Editor menu
 * Select **SpatialOS** > **Generate Code (force)** from the Unity Editor menu
 
-## Type mappings
+## Types
+
+### Type mappings
 
 The code generator maps from the [schema primitive types](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/schema/reference#primitive-types) to C# types according to the following table:
 
-| Schemalang type                | C# type      |
-| ------------------------------ | :---------------------: |
-| `int32` / `sint32` / `fixed32` | `int`                   |
-| `uint32`                       | `uint`                  |
-| `int64` / `sint64`/ `fixed64`  | `long`                  |
-| `uint64`                       | `ulong`                 |
-| `float`                        | `float`                 |
-| `bool`                         | `BlittableBool`         |
-| `string`                       | `string`                |
-| `bytes`                        | `byte[]`                |
-| `EntityId`                     | `Improbable.Gdk.Core.EntityId` |
+| Schemalang type                |               C# type                |
```

Comment:
nice! 🎉

Reply:
I'd recommend this vscode plugin: https://marketplace.visualstudio.com/items?itemName=fcrespo82.markdown-table-formatter